src/generate/genePackageJson.ts

```diff
@@ -69,8 +69,8 @@ export function genePackageJson (packageJsonPath: string, astParsingResult?: Ast
 
 export function getGreaterVersion (versionA: string, versionB: string): string {
   // only compare minimal version because it is hard to compare semantic version
-  const minVersionA = minVersion(versionA).version
-  const minVersionB = minVersion(versionB).version
+  const minVersionA = minVersion(versionA || '0.0.1').version
+  const minVersionB = minVersion(versionB || '0.0.1').version
```

Comment:
```suggestion
  const minVersionA = minVersion(versionA || '').version
  const minVersionB = minVersion(versionB || '').version
```
passing in `''` results `0.0.0` that lower than `0.0.1`